### PR TITLE
fix(fb_pixel): eslint issue

### PR DESCRIPTION
**Pull request by Gauravudia**
## PR Description

Please include a summary of the change along with the relevant motivation and context.

## Notion ticket

Ticket link

## Screenshots

Please add screenshots for any new features or UI bug fixes for the following browsers -

- Chrome
  >
- Firefox
  >
- Safari
  >

## Security

- [ ] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

### CODE REVIEW

File: src/integrations/FacebookPixel/browser.js [hunk 1]
```diff
@@ -334,9 +333,8 @@ class FacebookPixel {
       const contentType = this.getContentType(rudderElement, 'product');
       const contentIds = [];
       const contents = [];
-      if (products) {
-        for (let i = 0; i < products.length; i += 1) {
-          const product = products[i];
+      if (products && Array.isArray(products)) {
```
Comment (saikumarrs) [inline src/integrations/FacebookPixel/browser.js:336]:
```suggestion
      if (Array.isArray(products)) {
```

File: src/integrations/FacebookPixel/browser.js [hunk 2]
```diff
@@ -437,9 +435,8 @@ class FacebookPixel {
       let contentCategory = category;
       const contentIds = [];
       const contents = [];
-      if (products) {
-        for (let i = 0; i < products.length; i += 1) {
-          const product = products[i];
+      if (products && Array.isArray(products)) {
```
Comment (saikumarrs) [inline src/integrations/FacebookPixel/browser.js:438]:
```suggestion
      if (Array.isArray(products)) {
```

File: src/integrations/FacebookPixel/browser.js (hunk 1 above)
Comment (Gauravudia) [reply src/integrations/FacebookPixel/browser.js]: updated

File: src/integrations/FacebookPixel/browser.js (hunk 2 above)
Comment (Gauravudia) [reply src/integrations/FacebookPixel/browser.js]: updated

**Review by saikumarrs — APPROVED** [review]

**Pull request merged**